packages/core-components/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -83,16 +106,36 @@ export function ErrorPage(props: IErrorPageProps) {
         <Typography variant="h2" className={classes.title}>
           Looks like someone dropped the mic!
         </Typography>
-        <Typography variant="h6">
+        <Typography variant="h6" className={classes.goBackTitle}>
           <Link to="#" data-testid="go-back-link" onClick={() => navigate(-1)}>
             Go back
           </Link>
           ... or please{' '}
           <Link to={supportUrl || support.url}>contact support</Link> if you
           think this is a bug.
         </Typography>
+        {stack && (
+          <WarningPanel severity="info" subTitle="Stack Trace">
+            <Grid container>
+              <Grid item xs={11}>
+                <Typography
```

Comment:
Does the `CodeSnippet` component work as an alternative here? Would be nice to avoid a lot of the styling going on here

Reply:
Yes, that makes sense I can include the CodeSnippet as the message prop to the WarningPanel